docs/notebooks/keras_wrapper.ipynb

```diff
@@ -0,0 +1,800 @@
+      "Epoch 5/5\n",
+      "137/137 [==============================] - 2s - loss: 0.9002 - acc: 0.4891     \n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<keras.callbacks.History at 0x7f9b96fe95d0>"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sequence_input = Input(shape=(MAX_SEQUENCE_LENGTH,), dtype='int32')\n",
+    "embedded_sequences = embedding_layer(sequence_input)\n",
+    "x = Conv1D(128, 5, activation='relu')(embedded_sequences)\n",
+    "x = MaxPooling1D(5)(x)\n",
+    "x = Conv1D(128, 5, activation='relu')(x)\n",
+    "x = MaxPooling1D(5)(x)\n",
+    "x = Conv1D(128, 5, activation='relu')(x)\n",
+    "x = MaxPooling1D(35)(x)  # global max pooling\n",
+    "x = Flatten()(x)\n",
+    "x = Dense(128, activation='relu')(x)\n",
+    "preds = Dense(y_train.shape[1], activation='softmax')(x)\n",
+    "\n",
+    "model = Model(sequence_input, preds)\n",
+    "model.compile(loss='categorical_crossentropy', optimizer='rmsprop', metrics=['acc'])\n",
+    "\n",
+    "model.fit(x_train, y_train, epochs=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As can be seen from the results above, the accuracy obtained is not that high. This is because of the small size of training data used and we could expect to obtain better accuracy for training data of larger size."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "#### Integration with Keras : Another classification task"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this task, we train our model to predict the category of the input text. We start by importing the relevant modules and libraries : "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from keras.models import Sequential\n",
+    "from keras.layers import Dropout\n",
+    "from keras.regularizers import l2\n",
+    "from keras.models import Model\n",
+    "from keras.engine import Input\n",
+    "from keras.preprocessing.sequence import pad_sequences\n",
+    "from keras.preprocessing.text import Tokenizer\n",
+    "from gensim.models import keyedvectors\n",
+    "from collections import defaultdict\n",
+    "\n",
+    "import pandas as pd\n",
+    "import spacy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now define some global variables and utility functions which would be used in the code further : "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# global variables\n",
+    "\n",
+    "nb_filters=1200  # number of filters\n",
```

Comment:
PEP8: space around the assignment operator (`x = 1`).

Here and elsewhere in the notebook.